process/README.md

```diff
@@ -0,0 +1,22 @@
+# TAC Process Documentation #
+
+This folder contains the policies and procedures the OpenSSF TAC uses to perform their duties as well as guidelines for the Technical Initiatives of the Foundation.
+
+## TAC Election Process ##
```

Comment:
I think we should move this section down in the doc. From a TI's perspective, the processes I'd want to find most quickly are the Decision and Lifecycle sections.